docs/development/setup_development.rst

```diff
@@ -212,10 +214,9 @@ Install the dependencies for the development environment:
 #. Ansible_
 #. rsync >= 3.1.0
 
-.. note:: Note that the version of rsync installed by default on macOS is
-          extremely out-of-date, as is Apple's custom. We recommend using
-          Homebrew_ to install a modern version (3.1.0 or greater):
-          ``brew install rsync``.
+.. note:: Starting with OSX 10.13.3, you will need to `approve third-party apps`_ downloaded from the internet in your macOS Security & Privacy preferences. 
```

Comment:
For which application was this required?

Reply:
The default behavior in macOS 10.13 "High Sierra" allows for apps to be installed from outside the App Store, provided they are signed with a Apple Developer ID. This doesn't apply to the CLI so `brew` should be fine. So should be Vagrant and VirtualBox, iirc.